This is a bit tricky code patch
 - SBCL uses arm64 features on Apple Sillicon
 - ECL-21.2.1 uses arm
 - ECL-devel uses aarch64

See: https://gitlab.com/embeddable-common-lisp/ecl/-/issues/713

---
 libffi/libffi-types.lisp | 1 +
 1 file changed, 1 insertion(+)

diff --git libffi/libffi-types.lisp libffi/libffi-types.lisp
index b2ec1dd..b8374fe 100644
--- libffi/libffi-types.lisp
+++ libffi/libffi-types.lisp
@@ -73,6 +73,7 @@
  ((:default-abi "FFI_DEFAULT_ABI"))
  #-x86-64
  ((:sysv "FFI_SYSV"))
+ #-(or arm64 arm aarch64)
  ((:unix64 "FFI_UNIX64")))
 
 (ctype ffi-abi "ffi_abi")
-- 
2.40.1

